paperwork-backend/paperwork_backend/shell.py

```diff
@@ -520,22 +519,38 @@ def _do_import(filepaths, dsearch, doc, ocr=None, ocr_lang=None,
 
     for doc in import_result.new_docs:
         if guess_labels:
-            labels = dsearch.guess_labels(doc)
+            labels_ = dsearch.guess_labels(doc)
             r['guessed_labels'].append(
                 {
                     "docid": doc.docid,
-                    "labels": [label.name for (label, scores) in labels],
+                    "labels": [label.name for (label, scores) in labels_],
                 }
             )
-            for (label, scores) in labels:
+            for (label, scores) in labels_:
                 dsearch.add_label(doc, label, update_index=False)
+        for label_name, color in labels:
+            label = None
+            for clabel in dsearch.label_list:
+                if clabel.name == label_name:
+                    label = clabel
+                    break
+            if not label:
```

Comment:
You can use `for ... else` here:

```
for clabel in dsearch.label_list:
    if clabel.name == label_name:
        label = clabel
        break
else:
    (...)
```
( http://book.pythontips.com/en/latest/for_-_else.html )

Reply:
Thanks, I don't know this, nice :-)